tests/integration/components/grid-stack-test.js

```diff
@@ -24,8 +24,9 @@ module('Integration | Component | grid stack', function(hooks) {
       </GridStack>
     `);
 
-    assert.ok(
-      this.$('.grid-stack .grid-stack-item').is('.ui-draggable.ui-resizable'),
+    assert.strictEqual(
```

Comment:
Can we update this to use qunit-dom? 
https://github.com/simplabs/qunit-dom/blob/master/API.md#hasclass

Reply:
Updated!